### DISCUSSION TITLE
Unterschied Victron Leistung zwischen 2P2S vs 4S

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#429] [Q&A]
Hallo @spcqike

I habe eine Frage:
Ich teste eine Pylontech US5000, einen Victron 150/35 und 4 Paneele (ASTRO 5s 405Wp)
Rated voltage (Vmpp / V) 31.26 
Rated current (Impp /A) 12.96
Open circuit voltage (Voc / V)  37.20
Short circuit current (Isc /A)13.76

Mit einer 2P2S Kombination gibt der Victron eine Max Leistung von 1280W
Mit einer 4S Kombination aber, erreicht der Victron fast 1700W

Ist es logisch, oder mache ich irgendwo einen Fehler?

**Comment by spcqike** [comment]
Erscheint mir nicht logisch. Vielleicht regelt der MPPT da einfach sehr ungünstig aus. 

Sind die Paneele alle gleich ausgerichtet? 

Mir erschließt sich aber auch nicht, dass die Paneele 110% liefern bei den aktuellen Temperaturen :) (1700W ladeleistung bei 1620Wp modulleistung, unter Berücksichtigung der ~95% Effizienz des ladereglers)

Sicher, dass da 1.700W raus kommen? 

1.280W nach dem laderegler wären so 1350W vorher. Also 83% des Modul ratings. Klingt für mich eigentlich sinnig.

**Reply by Manos1966** [reply]
OK, Danke! Ich werde anfangen zu testen.
alle Paneele  sind gleich ausgerichtet.

Ich habe oefters Situationen erlebt, wo der Victron eine hoehere Leistung erreicht hat als die nominale Leistung der Paneele.

**Reply by Manos1966** [reply]
Ich glaube das Problem verstanden zu haben!

Ich habe in der Vergangenheit ohne Solar Pass-through getestet.

Seitdem die Pylotech angeschlossen wurde, teste ich mit Solar Pass-through.
...und habe folgendes festgestellt:
Die Battery hat das Limit 96% fuer *Akku SOC - Start solar passthrough* gegen 14:30 erreicht
![Victron LeistungA](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/123ff843-edda-474c-af42-46fc742bdf53)
...und danach blieb konstant auf 96%
![Victron LeistungAb](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/c462976c-7fee-421c-8b75-68cd67c7ca06)

Danach war ich verwirrt, weil der Victron kein Strom produziert hat
![Victron LeistungC](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/350a0d28-19a1-4e89-ad01-708c893fa1b3)
Vielleicht hat ja der Victron geregelt, weil die Batterie voll war.... aber dies konnte nicht sein, da die Batterie bei 96% war.

Und dann habe ich bemerkt, dass der Hoymiles angefangen hat, Strom zu produzieren! Also Solar Pass-through funktioniert.
![Victron LeistungB](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/31c97402-d738-4891-ac6a-2b430367c34a)

Daraufhin habe ich den Batterie Ladestand % überprüft und siehe da, während der Hoymiles Strom produziert hat, ist der Ladezustand constant auf 96% geblieben!
Letzteres bedeutet,  der Hoymiles hat den Strom des Victrons genommen, daher die Werte die zeigen dass der Victron nicht mehr produziert hat, **sind falsch!**
Die Victron Leistung die zur Batterie geht wird errechnet (Volt x Amp). 
![Victron LeistungD](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/a353d1c4-844a-4b8c-862a-876ed1329cc9)

Warum jetzt wenn der Hoymiles die gleiche Leistung abzieht wie der Victron produziert, die Amperes null werden (und dadurch die Victron Leistung Volt x Amp) null wird, weiss ich nicht.  
Strikt gesehen es ist sogar richtig dass es Null zeigt, weil laut Wiki, es handelt sich um "Victrons Battery Output". Es stimmt, nichts geht an die Batterie :-d

Wie auch immer, ich glaube ich habe herausgefunden warum die Leistung meines Victron niedriger ist heutzutage.

**Reply by madmartin** [reply]
>Vielleicht hat ja der Victron geregelt, weil die Batterie voll war.... aber dies konnte nicht sein, da die Batterie bei 96% war.

Diese Schlussfolgerung ist gewagt. Schau Dir das Feld "State of operation" vom Lader an, da steht ob er lädt oder nicht. Und den "State of Operation" entscheiden die konfigurierten Spannungswerte und der zeitliche Verlauf der Batteriespannung. Von den Prozentwerden des BMS hat der Lader keine Ahnung.
Daher würde ich mir das Ladeprofil vom Victron Lader nochmal ansehen.

**Reply by Manos1966** [reply]
Hallo @madmartin 

du hast Recht! 
Ich habe Solar Pass-through bei 60% aktiviert und konnte sofort den Fehler finden: 

Ich habe bis jetzt den falschen MQTT-Wert fuer meine Berechnung der Victron Leistung benutzt :-(
Anstatt  VictronLeistung = **V x I** zu berechnen
<html>
<body>


victron/[serial]/V | R | Voltage | Volt (V)
-- | -- | -- | --
victron/[serial]/I | R | Current | Ampere (A)


</body>
</html>

habe ich VictronLeistung = **V x current**
und "current" ist die aktuelle Leistung der Pylontech (die beim aktivierten Solar Pass-through Null sein wird)  <html>
<body>


battery/current | R | Actual current | Ampere (A)
-- | -- | -- | --



</body>
</html>

![Victron LeistungE](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/794c00c6-28c1-46a0-8896-dcc018e76e41)

Vielen Dank!

**Comment by Manos1966** [comment]
Hallo @madmartin @spcqike 

ich sehe weiterhin einen Unterschied der Victron Leistung zwischen 2P-2S und 4S
2P-2S hat eine Max Leistung von 1400W
4S dagegen, 1755W

Alle vier Paneele sind gleich ausgerichtet.
Victron 150/35 und 4 Paneele (ASTRO 5s 405Wp)

Merkwurdig
![4S 1700W](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/debbd9e8-df67-4495-a49a-52e78d795852)


![2S2P](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/e7e53b4e-7444-447f-8944-d48140dfeca4)

**Reply by JaMo523** [reply]
Wie ist denn deine Leitungslänge + Leitungsquerschnitt von Solarmodul bis Victron?

**Reply by Manos1966** [reply]
6mm2 
Leitungslaenge muss ich messen....

**Reply by Manos1966** [reply]
Hmmm.... die Leitungen sind doch laenger als ich dachte... 20 meter von den Paneelen bis zum Victron!

**Reply by spcqike** [reply]
Wie ist der Aufbau/die Verkabelung bei 2S-2P? nutzt du Y-Adapter an den Paneelen, oder hast du eine Leitung pro String zum Laderegler? 

300W Verlust durch lange Leitungen und Y-Adapter sind nicht unmöglich. Unter der Annahme, dass die Paneele zur Spitze ihre 12.86A liefern, bei 2S-2P also 25.72A, braucht es für 300W Verlustleistung "nur" 0.45Ohm Gesamtwiderstand.

auf der anderen Seite, wie verlässlich sind deine Daten? Die Paneele sollten eine VoC von 37V haben, oder? Wie kommst du da bei 2S-2P auf 118V Stringspannung?

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/a7969c71-06ae-47a0-a186-1b7ff465da2d)

**Reply by JaMo523** [reply]
Das mit den 118V habe ich mich auch schon gefragt. 20m ein Weg, also 40m insgesamt (hin und zurück)?

**Comment by Manos1966** [comment]
@spcqike ich (hatte) Y-Adapter an den Paneelen genutzt. Ja, es sind 20hin+20zurueck = 40 meter 

> 300W Verlust durch lange Leitungen und Y-Adapter sind nicht unmöglich. Unter der Annahme, dass die Paneele zur Spitze ihre 12.86A liefern, bei 2S-2P also 25.72A, braucht es für 300W Verlustleistung "nur" 0.45Ohm Gesamtwiderstand.

WOW!

Wegen des Bildes... 118.91V macht kein Sinn, ich muss meine gespeicherte Bilder pruefen. Ich kann sein, dass ich das falsche Bild benutzt habe. Meine Sorge war die Leistung die nie hoeher als 1400W bei 2P-2S ging, dadurch habe ich nur auf die 1400W Bild Suche....

Noch beunruhinger finde ich deine Bestaetigung, 300W Verlust waere sogar moeglich :-(
Ich muss mein Schulwissen ueber das Ohmische Gesetzt auffrischen...   
Bis jetzt waren die Paneele auf dem Boden, der Victron direkt daneben, die Leitungen waren also 1 meter lang.

**Reply by spcqike** [reply]
> Ich muss mein Schulwissen ueber das Ohmische Gesetzt auffrischen...

so schwer ist das nicht. 
Spannung(U) ist Widerstand (R) mal Strom (I)
Leistung (P) ist Spannung (U) mal Strom (I)

also ist Leistung (P) gleich Widerstand (R) mal Strom zum Quadrat (I^2)

bei gewollten Verbrauchern eben Leistung. Bei ungewollten eben Verlustleistung. 

300W bei 25.72A sind dann halt 0.45Ohm. 

Als Faustregel kann man 0.1Ohm pro Verbindung hernehmen. Daher ist es wichtig auf guten Sitz / Verbindung zu achten. Der Widerstand im Kabel ist in der Regel kleiner. 

Diverse online Rechner kommen bei deinem setup (1.600Wp, 20m, 72V) auf 12mm^2 Querschnitt. Also könnten auch deine 6mm^2 hinter dem Y-Adapter zu gering sein. 

(Da sie auch zu dem Ergebnis kommen, dass du mit deinen 6mm^2 ~3V Spannungsabfall hast, was ja wieder ~75W entsprechen, solltest du die Verkabelung wirklich prüfen. Wenn du 2S2P machen willst)

**Reply by Manos1966** [reply]
Das Thema 2S2P ist vorbei, da die Reihe bessere Ergebnisse bringt.
Aber jetzt verstehe ich aber auch warum...
Zu einem, der Victron sollte in der Naehe der Batterie sein (da seine Spanung von 52V kleiner ist, als die 72V Spanung der 2P2S Module, bzw die 120V der 4S Module).
Zu zweitem, lieber 4S (hoehere Spanung) als 2P2S, da weniger Verluste. So lange der Victron die Spannung ertragen kann, natuerlich.

**Reply by JaMo523** [reply]
Für deine Anlage stimmt die Aussage. Ergänzen kann man, dass bei zwei verschiedenen Modul Ausrichtungen 2S2P mehr Sinn macht.

**Comment by Manos1966** [comment]
OK, wenn ich diese Berechnungen des notwendigen Querschnitts sehe, dann brauche ich nicht weiter suchen....

![4S vs 2P2S Kabelquerschnitt](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/e53f2863-b0e5-45d9-b481-7a2967549442)

**Reply by spcqike** [reply]
Wie gesagt, zu viele und vor allem günstige steckverbinder sind auch schlecht. 

Lieber 20m 4x6qmm als 1m 4x6qmm, 2 y-Adapter und dann 19m 2x12qmm.

**Reply by madmartin** [reply]
Willst Du damit sagen daß anstelle der Y-Adapter lieber die Kabel parallel verlegt werden sollen und dann erst direkt beim Victron Lader zusammenkommen?
Je nach Querschnitt stelle ich mir das etwas eng an den Anschlussklemmen vor...

**Reply by JaMo523** [reply]
Also man kann z.b einen Klemmblock von einer Hauptverteilung oder eine Sammelschiene nutzen. Wenn dann das letzte Stück (0,5m) in 6-16 mm²  von Klemmblock bis Anschlussklemme Victron verlegt ist, dann sollte es nicht so schlimm sein.

**Reply by madmartin** [reply]
> ... Klemmblock von einer Hauptverteilung ...

Meinst Du sowas hier: ?

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/4208120/486f0c20-47d5-49d8-8631-42b32b351744)

Da sehe ich das Problem daß man bei dieser Art Klemmen die Litzen nicht ohne Aderendhülse reinstecken darf, und die Aderendhülse bringt hier wieder zusätzlichen Übergangswiderstand.

**Reply by JaMo523** [reply]
Ob mit oder ohne Aderendhülse kommt auf den Aufbau der Klemme an. Wenn man sich unsicher ist kann man sich das Datenblatt vom Hersteller ansehen. Da muss angegeben werden ob eine Aderendhülse verwendet werden muss.


Sehe das verwenden einer Aderendhülse hier auch nicht als Problem weil:

1. Bei der Verbindung ist nicht nur die Kontaktfläche sondern auch der Kontaktdruck entscheidend. Eine Aderendhülse würde ich nicht als Übergangswiderstand bezeichnen.
2.  Wie reden bei 120V von einem Strom zwischen 10-20A.

Ansonsten gibt es immer noch die Möglichkeit Sammelschiene + Klemme oder Kabelschuh.

**Comment by Manos1966** [comment]
Ich habe sehr viel gelernt!!!

Vielen Dank Leute 👍
